modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java

```diff
@@ -528,6 +529,25 @@ public int getObjectDefinitionsCount(long companyId)
 		return objectDefinitionPersistence.countByCompanyId(companyId);
 	}
 
+	@Override
+	public ObjectDefinition getSystemObjectDefinition(
+			long companyId, String name)
+		throws PortalException {
+
+		for (ObjectDefinition systemObjectDefinition :
+				getSystemObjectDefinitions()) {
+
+			if (Objects.equals(name, systemObjectDefinition.getName()) &&
+				Objects.equals(
+					companyId, systemObjectDefinition.getCompanyId())) {
+
+				return systemObjectDefinition;
+			}
+		}
+
+		throw new NoSuchObjectDefinitionException();
+	}
```

Comment:
@LuismiBarcos can we use the existing method [fetchObjectDefinition](https://github.com/liferay/liferay-portal/blob/b02120e30d0c5ef2493e3537a7b1ab954afb5671/modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java#L481-L484) which fetches by `companyId` and `name`?

Reply:
Mmmmm, I didn't want to use it because I was able to create a custom object with the name `User` exactly the same name as the User system object. I don't know if this is the expected behavior.
![image](https://user-images.githubusercontent.com/17163902/225664267-1109873e-cd23-4ce6-a02c-a59488ed5a92.png)

This made me doubt if this method was only for custom objects or also applies to system objects too.

Rethinking it now and checking the DB, I see that the names stored are different for the system objects and the custom objects (I completely forgot about the `C_` prefix that is added for custom objects 😅 )

So, I will rework this to use the method you point out and avoid adding this useless code that I added in this PR to the object-service. 

Thanks for the review and my apologies for the inconvenience. 
I've should know that 😅 

I will close this PR

Reply:
No problem, @LuismiBarcos

Thanks for working on this 😄